spec/acceptance/test_unit_spec.rb

```diff
@@ -4,6 +4,16 @@
 describe 'pdk test unit', module_command: true do
   include_context 'with a fake TTY'
 
+  shared_context 'with spec file' do |filename, content|
+    around(:all) do |example|
+      path = File.join('spec', 'unit', filename)
+      FileUtils.mkdir_p(File.dirname(path))
+      File.open(path, 'w') { |f| f.puts content }
```

Comment:
Do we care about line endings here?  (not `wb` so on Windows it'll use CRLF)

Reply:
Nope, for the purposes of these short lived files, the line endings don't matter